external/esp_idf_port/wpa_supplicant/include/wpa2/tls/libtommath.h

```diff
@@ -43,13 +43,13 @@ static const char mem_debug_file[] ICACHE_RODATA_ATTR = __FILE__;
 #endif
 
 #define BN_MP_INVMOD_C
-#define BN_S_MP_EXPTMOD_C /* Note: #undef in tommath_superclass.h; this would
-			   * require BN_MP_EXPTMOD_FAST_C instead */
+#define BN_S_MP_EXPTMOD_C /* Note: #undef in tommath_superclass.h; this would \
```

Comment:
I am really sorry for your effort. But we don't manage external folder and we want to keep it as original. I think we had better exclude the change on external if there is no problem in operation.

Reply:
It's okay. It was an internal problem on my part.